include/irr/asset/filters/dithering/IDither.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2020 - AnastaZIuk
+// This file is part of the "IrrlichtBAW" engine.
+// For conditions of distribution and use, see copyright notice in irrlicht.h
+
+#ifndef __IRR_I_DITHER_H_INCLUDED__
+#define __IRR_I_DITHER_H_INCLUDED__
+
+#include "irr/core/core.h"
+#include "irr/asset/ICPUImage.h"
+#include "irr/asset/ICPUImageView.h"
+
+namespace irr
+{
+    namespace asset
+    {
+        class IDither
+        {
+            public:
+                virtual ~IDither() {}
+                virtual float pGet(const core::vectorSIMDu32& pixelCoord) = 0;
```

Comment:
you need a pointer to `IState` as another parameter

Reply:
solved